BAC0/core/devices/Points.py

```diff
@@ -1187,8 +1183,8 @@ def __init__(self, device, name):
 
         self.properties.description = props["description"]
         self.properties.units_state = props["units_state"]
-        self.properties.simulated = "Offline"
-        self.properties.overridden = "Offline"
+        self.properties.simulated = (True, None)
```

Comment:
I looked and didn't see anywhere checking for these strings, and they seem to conflict with the usual type of `Tuple[bool, Optional[int]]`